eng/common/scripts/Helpers/Resource-Helpers.ps1

```diff
@@ -204,3 +204,82 @@ function Wait-PurgeableResourceJob {
     }
   }
 }
+
+# Helper function for removing storage accounts with WORM that sometimes get leaked from live tests not set up to clean
+# up their resource policies
+function Remove-WormStorageAccounts() {
```

Comment:
This is moved from an independent script. Instead of running that script to clean up storage accounts, we can just run `Remove-TestResources.ps1` instead.